src/components/Proposals/ProposalCard.js

```diff
@@ -1,25 +1,75 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Card, GU, textStyle, useTheme } from '@aragon/ui'
-import { DISPUTABLE_VOTE_STATUSES } from './disputable-vote-statuses'
+import { AppBadge, Card, GU, textStyle, useTheme } from '@aragon/ui'
+import {
+  DISPUTABLE_VOTE_STATUSES,
+  VOTE_STATUS_CANCELLED,
+  VOTE_STATUS_DISPUTED,
+  VOTE_STATUS_PAUSED,
+} from './disputable-vote-statuses'
 import ProposalOption from './ProposalOption'
 import DisputableStatusLabel from './DisputableStatusLabel'
 
-function ProposalCard({ vote, onProposalClick }) {
+function getAttributes(status, theme) {
+  const attributes = {
+    [VOTE_STATUS_CANCELLED]: {
+      backgroundColor: theme.surfacePressed,
+      borderColor: theme.controlUnder,
+      disabledProgressBars: true,
+    },
+    [VOTE_STATUS_PAUSED]: {
+      backgroundColor: '#fffdfa',
+      borderColor: theme.warning,
+      disabledProgressBars: true,
+    },
+    [VOTE_STATUS_DISPUTED]: {
+      backgroundColor: '#FFF7F2',
+      borderColor: '#D26C41',
+      disabledProgressBars: true,
+    },
+  }
+
+  return (
+    attributes[status] || {
+      backgroundColor: theme.surface,
+      borderColor: theme.border,
+      disabledProgressBars: false,
+    }
+  )
+}
+
+function ProposalCard({ appAddress, vote, onProposalClick }) {
   const theme = useTheme()
   const { context, voteId } = vote
 
+  const disputableStatus = DISPUTABLE_VOTE_STATUSES.get(vote.status)
+  const { backgroundColor, borderColor, disabledProgressBars } = getAttributes(
+    disputableStatus,
+    theme
+  )
+
   return (
     <Card
       onClick={() => onProposalClick(voteId)}
       css={`
         display: grid;
         grid-template-columns: 100%;
-        grid-template-rows: 1fr auto auto;
+        grid-template-rows: auto 1fr auto auto;
         grid-gap: ${1 * GU}px;
         padding: ${3 * GU}px;
+        background: ${backgroundColor};
+        border: solid 1px ${borderColor};
       `}
     >
+      <div
+        css={`
+          display: flex;
+          justify-content: space-between;
```

Comment:
I have a feeling we don't actually need these flex rules as it should just sit at the start of the container without them. If not we can also use `justify-content: flex-start` to position it at the very beginning (which I think is the default for flex anyway).